web/client/epics/layers.js

```diff
@@ -168,8 +170,41 @@ export const updateSettingsParamsEpic = (action$, store) =>
                 Rx.Observable.empty());
         });
 
+/**
+ * Sets control properties to currently selected group when catalogue is open
+ * Sets the currently selected group as the detination of new layers in catalogue
+ * Keeps a record of previously selected group to set the groupId in setControlProperties to default
+ * if the current group is deselected it resets the groupId to Default
+ *  Action performed: setControlProperties (only if nodeType is group, catalogue is open)
+ * @memberof epics.layers
+ * @param {external:Observable} action$ manages `SELECT_NODE`
+ * @return {external:Observable}
+ */
+export const updateGroupSelectedMetadataExplorerEpic = (action$, store) => {
```

Comment:
I think this epic should be moved in the catalog related epic file. This behaviour should work only if the catalog plugin is included